locales/ar_SA/pcsx2_Main.po

```diff
@@ -5,28 +5,29 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: PCSX2 1.0\n"
+"Project-Id-Version: PCSX2 1.6\n"
```

Comment:
1.6.0

Reply:
Thats weird. It doesnt say that with mines. Maybe i committed it to quickly.

Reply:
https://github.com/RedDevilus/pcsx2/blob/RedDevilus-ML-Localisations-3/locales/ar_SA/pcsx2_Main.po

Reply:
Forgot the 0.